src/crypto/index.ts

```diff
@@ -1302,7 +1302,8 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
         // make sure we have a Uint8Array, rather than a string
         if (typeof encodedKey === "string") {
             key = new Uint8Array(olmlib.decodeBase64(fixBackupKey(encodedKey) || encodedKey));
-            await this.storeSessionBackupPrivateKey(key);
+            // legacy crypto doesn't care about the version
+            await this.storeSessionBackupPrivateKey(key, "");
```

Comment:
yes. I don't think we need to change legacy crypto's behaviour at all here? It should continue to accept callls to `storeSessionBackupPrivateKey` with `version=undefined`.